articles/configure-your-rich-text-editors.mdx

```diff
@@ -7,390 +7,153 @@ nav_title: Configure your rich text editors
-    Enables a control to copy formatting from text to other text. Only applies to formatting from `bold`, `italic`, `underline`, `strike`, `subscript`, and `superscript`. Does not copy other styles or formatting.
-
-    Available for `_editables.text`, `_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-</comp.DataReference>
-
-### Paths
-
-These paths control where new asset files are uploaded to. They also set the default path when choosing existing images, and linking to existing files.
-
-<comp.DataReference>
-  <comp.DataReferenceRow label="paths" type_markdown="Object">
-    Paths to common folders. Each path is relative to [global](/documentation/articles/create-a-global-configuration-file/) site `source`. If unset, the paths default to those configured in the [global configuration](/documentation/articles/create-a-global-configuration-file/). The available keys are:
-
-    * `uploads` for the default location of newly uploaded site files.
-    * `dam_uploads` for the default location of newly uploaded [DAM](/documentation/articles/introduction-to-assets-and-dams/) files.
-    * `dam_static` for the location of statically copied assets for [DAM](/documentation/articles/introduction-to-assets-and-dams/) files.
-
-    `uploads` and `dam_uploads` have a [number of dynamic placeholders available](/documentation/articles/adjusting-the-uploads-path/).
-
-    Available for `_editables.link`, `_editables.text`, `_editables.image`,`_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-</comp.DataReference>
-
-### Styles and alignment
-
-Add predefined styles in plain CSS for your clients and team members to use in the Visual Editor and Content Editor.
-
-<comp.DataReference>
-  <comp.DataReferenceRow label="styles" type_markdown="String">
-    Enables a drop down menu for editors to style selected text or blocks or text. Styles are the combination of an element and class name. The value for this option is the path (either source or build output) to the CSS file containing the styles.
+_editables___1___:
+  content___2___:
+    bold: true
+    italics: true
+    underline: true
 
-    Available for `_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="left" type_markdown="String">
-    Enables a control to left align text by toggling a class name for a block of text. The value is the class name the editor should add to align the text. The styles for this class need to be listed in the `styles` file to take effect outside of the Visual Editor.
+```
+<comp.Annotation number="1">
+  Configure the Content Editor and all editable regions under the `_editables` key anywhere in the [configuration cascade](/documentation/articles/using-the-configuration-cascade/).
+</comp.Annotation>
+<comp.Annotation number="2">
+  Configuration options for the Content Editor are nested keys under `content`.
+</comp.Annotation>
+</comp.MultiCodeBlock>
```

Comment:
This would be another great place (after the first code sample) to mention that after configuring these, these will be the _only_ three options in the toolbar, all other defaults will be removed.